terraform/modules/hub/files/tasks/hub-config.json

```diff
@@ -36,20 +36,31 @@
       "server",
       "/tmp/config.yml"
     ],
-    "environment": [{
-      "Name": "CONFIG_DATA_PATH",
-      "Value": "/tmp/federation/configuration/config-service-data/${deployment}"
-    }, {
-      "Name": "RP_TRUSTSTORE_PASSWORD",
-      "Value": "${truststore_password}"
-    }, {
-      "Name": "CLIENT_TRUSTSTORE_PASSWORD",
-      "Value": "${truststore_password}"
-    }, {
-      "Name": "DEPLOYMENT",
-      "Value": "${deployment}"
-    }, {
-      "Name": "JAVA_OPTS",
-      "Value": "-Dservice.name=config -Xms3000m -Xmx3200m -XX:+HeapDumpOnOutOfMemoryError -Dhttp.proxyHost=\"egress-proxy.${domain}\" -Dhttp.proxyPort=\"8080\" -Dhttps.proxyHost=\"egress-proxy.${domain}\" -Dhttps.proxyPort=\"8080\" -Dhttp.nonProxyHosts=\"\" -Dnetworkaddress.cache.ttl=5 -Dnetworkaddress.cache.negative.ttl=5" }]
+    "environment": [
+      {
+        "Name": "CONFIG_DATA_PATH",
+        "Value": "/tmp/federation/configuration/config-service-data/${deployment}"
+      },
+      {
+        "Name": "RP_TRUSTSTORE_PASSWORD",
+        "Value": "${truststore_password}"
+      },
+      {
+        "Name": "CLIENT_TRUSTSTORE_PASSWORD",
+        "Value": "${truststore_password}"
+      },
+      {
+        "Name": "DEPLOYMENT",
+        "Value": "${deployment}"
+      },
+      {
+        "Name": "JAVA_OPTS",
+        "Value": "-Dservice.name=config -Xms3000m -Xmx3200m -XX:+HeapDumpOnOutOfMemoryError -Dhttp.proxyHost=\"egress-proxy.${domain}\" -Dhttp.proxyPort=\"8080\" -Dhttps.proxyHost=\"egress-proxy.${domain}\" -Dhttps.proxyPort=\"8080\" -Dhttp.nonProxyHosts=\"\" -Dnetworkaddress.cache.ttl=5 -Dnetworkaddress.cache.negative.ttl=5",
+      },
+      {
+        "Name": "PROMETHEUS_METRICS_PATH",
```

Comment:
What's this environment variable for? Who consumes it? Note that `engineering.reliability.gds.metrics.config.Configuration` (the only place I could find which uses this envvar) isn't used by PrometheusBundle (only MetricsBundle which is the "old and busted" way of doing things).

I think we can get rid of all of these.

Reply:
Hub application consumes it.

From readme file from https://github.com/alphagov/gds_metrics_dropwizard#how-to-build-your-project,  it says 

By default, metrics will be exposed at the path /metrics. You can change this with an environment variable like this:

```
PROMETHEUS_METRICS_PATH=/metrics
```
Dropwizard already has this endpoint `/metrics` on the admin port. That's why I added the environment variable to use a different endpoint `/prometheus/metrics` for Prometheus metrics.

Reply:
I see Prometheus Bundle and Metrics Bundle are different. Could you please update the readme file in gds_metrics_dropwizard repo about this change.